engine/frontend/src/component/Home.js

```diff
@@ -40,9 +40,14 @@ const Home = () => {
 
     const handleDeleteClick = (enclaveName) => {
         const makeRequest = async () => {
-            const filteredEnclaves = enclaves.filter(enclave => enclave.name !== enclaveName)
-            await removeEnclave(appData.jwtToken, appData.apiHost, enclaveName)
-            setEnclaves(filteredEnclaves)
+            try {
+                const filteredEnclaves = enclaves.filter(enclave => enclave.name !== enclaveName)
+                await removeEnclave(appData.jwtToken, appData.apiHost, enclaveName)
```

Comment:
@Peeeekay doesn't removeEnclave return anything? THe endpoint must return an error code if it fails?

Reply:
No; the response failure normally are returned as errors - if you look at the proto schema it's just an empty response